File: app/Pizzaria.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Notifications\Notifiable;
use Illuminate\Foundation\Auth\User as Authenticatable;

class Pizzaria extends Authenticatable
{
  use Notifiable;
  // const CREATED_AT = 'date_created';
  // const UPDATED_AT = 'date_modified';
  protected $fillable = [
    "nome",
    "contacto",
    "email",
    "endereco",
    "imagem",
    "username",
    "password",
    "descricao",
    "created_at",
    "updated_at"
  ];

  /**
   * The attributes that should be hidden for arrays.
   *
   * @var array
   */
  protected $hidden = [
      'password', 'remember_token',
  ];

  function produtos(){
    return $this->hasMany(\App\Produto::class);
  }

  function encomendas(){
    return $this->hasMany(\App\Encomenda::class);
  }
}

File: android/app/src/main/java/app/pizzar/model/Pizzaria.java
package app.pizzar.model;

public class Pizzaria {
    public long id;
    public String nome;
    public String contacto;
    public String email;
    public String endereco;
    public String imagem;
    public String username;
    public String password;
    public String descricao;
    public String created_at;
    public String updated_at;

    @Override
    public String toString() {
        return nome;
    }
}
